migrations/2024-08-20-053740_add_sec_code_and_bank_type_to_payouts/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Your SQL goes here
+ALTER TABLE payouts
+ADD COLUMN sec_code VARCHAR NULL,
+ADD COLUMN bank_type VARCHAR NULL;
```

Comment:
Is it possible to restrict these columns to a fixed length, say 64 or 128 chars?